5.5/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN chmod 755 /*.sh
 # Exposed ENV
 ENV MYSQL_USER admin
 ENV MYSQL_PASS **Random**
+ENV ON_CREATE_DB **False**
```

Comment:
What's the purpose of setting a value that flags it to later unset it? Why not leave it undefined by default?